src/Cake.Testing/Cake.Testing.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Contains testing utilities for Cake.</Description>
+    <Copyright>Copyright (c) .NET Foundation and contributors</Copyright>
+    <VersionPrefix>0.20.0</VersionPrefix>
+    <Authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park and contributors</Authors>
```

Comment:
As per the PR that I raised, we should extend this to include Alistair and Martin.